packages/core/examples/stackAPI/index.ts

```diff
@@ -161,6 +161,41 @@ addButtonToToolbar({
   },
 });
 
+addButtonToToolbar({
+  title: 'Rotate Absolute 150',
+  onClick: () => {
+    // Get the rendering engine
+    const renderingEngine = getRenderingEngine(renderingEngineId);
+
+    // Get the stack viewport
+    const viewport = <Types.IStackViewport>(
+      renderingEngine.getViewport(viewportId)
+    );
+
+    viewport.setProperties({ rotation: 150 });
+
+    viewport.render();
+  },
+});
+
+addButtonToToolbar({
+  title: 'Rotate Delta 30',
```

Comment:
I added this button because it is most likely to be used by a third party app that just does incremental rotation.